app/src/main/java/org/projectbuendia/client/ui/chart/PatientChartActivity.java

```diff
@@ -271,6 +281,28 @@ public void onPageFinished(WebView view, String url) {
                 mController.onAddTestResultsPressed();
             }
         });
+
+        initChartMenu();
+    }
+
+    private void initChartMenu() {
+        List<Chart> charts = mController.getCharts();
+        String[] menuArray = new String[charts.size()];
+        int i = 0;
+        for (Chart chart : charts) {
+            menuArray[i] = chart.name;
+            i++;
+        }
+        mAdapter = new ArrayAdapter<String>(this, android.R.layout.simple_list_item_1, menuArray);
+        mDrawerList.setAdapter(mAdapter);
+        final Activity that = this;
```

Comment:
Ooops. Forgot to remove this line. 
I didn't know about the OuterClass.this. I tried to use it inside the bellow onItemClick and it says: "Cannot resolve symbol". What I'm doing wrong?